cibuildwheel/util.py

```diff
@@ -24,37 +26,37 @@ def get_build_verbosity_extra_flags(level):
 
 
 class BuildSelector:
-    def __init__(self, build_config, skip_config):
+    def __init__(self, build_config: str, skip_config: str):
         self.build_patterns = build_config.split()
         self.skip_patterns = skip_config.split()
 
-    def __call__(self, build_id):
-        def match_any(patterns):
+    def __call__(self, build_id: str) -> bool:
+        def match_any(patterns: List[str]) -> bool:
             return any(fnmatch(build_id, pattern) for pattern in patterns)
         return match_any(self.build_patterns) and not match_any(self.skip_patterns)
 
-    def __repr__(self):
+    def __repr__(self) -> str:
```

Comment:
unnecessary

Reply:
Seems necessary to shut up `mypy`? See above.